Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/PrivateRegistryImageSetting.java

```diff
@@ -69,11 +69,10 @@ public void setPassword(String password) {
         this.password = password;
     }
 
-    @Override
-    public String getImageNameWithTag() {
+    public String getImageTagWithServerUrl() {
         if (this.serverUrl != null && super.getImageNameWithTag() != null) {
```

Comment:
better to make member `protected` in super class

Reply:
will fix this in next PR